x-pack/plugins/cases/public/components/all_cases/all_cases_list.test.tsx

```diff
@@ -927,6 +911,20 @@ describe('AllCasesListGeneric', () => {
           expect(deleteCasesSpy).toHaveBeenCalledWith(['basic-case-id'], expect.anything());
         });
       });
+
+      it('should disable row actions when bulk selecting cases', async () => {
```

Comment:
Could we add a test for disabling the row actions when a single checkbox is selected?